<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RailsPointEditor</class>
 <widget class="QWidget" name="RailsPointEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Высота КС</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Наклон</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="tangSpin">
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="connectButt">
     <property name="text">
      <string>Режим соеденинения траекторий</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Вес</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="trajRemPButt">
     <property name="text">
      <string>Режим удаления точек</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="trajAddPButt">
     <property name="text">
      <string>Режим добавления точек</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Уклон</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="cheightSpin"/>
   </item>
   <item row="0" column="1">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="tiltSpin">
     <property name="minimum">
      <double>-45.000000000000000</double>
     </property>
     <property name="maximum">
      <double>45.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="activeButt">
     <property name="text">
      <string>Сделать активными точки</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
